framework/doc/content/source/actions/AddADDGKernelAction.md

```diff
@@ -0,0 +1,4 @@
+# AddADDGKernelAction
```

Comment:
I thought we talked about not having a separate action and instead performing casts to figure out the correct warehouse for depositing things? I know this isn't currently what is being done for the other AD object, but we need to switch those over

Reply:
OK, I am going to make this work with the old `DGKernel` block.